<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" parallel="none">
  <parameter name="testurl" value="http://192.168.8.34:8080/4d_width"/>
  <parameter name="timeOut" value="20"/>
  <parameter name="browserName" value="chrome"/>
  <parameter name="sleepTime" value="800"/>
  <parameter name="waitMillisecondsForAlert" value="40000"/>
  <listeners>
    <listener class-name="com.blueway.test.uit.plugins.TestResultListener"/>
    <listener class-name="com.blueway.test.uit.plugins.RetryListener"/>
    <listener class-name="com.blueway.test.uit.plugins.PowerEmailableReporter"/>
  </listeners>
  <test name="Login(failed)" parallel="none">
    <classes>
      <class name="com.blueway.test.uit.testcase.login.Login_001">
        <methods>
          <include name="startTest"/>
          <include name="endTest"/>
        </methods>
      </class> <!-- com.blueway.test.uit.testcase.login.Login_001 -->
      <class name="com.blueway.test.uit.testcase.login.Login_002">
        <methods>
          <include name="login" invocation-numbers="3"/>
          <include name="startTest"/>
          <include name="endTest"/>
        </methods>
      </class> <!-- com.blueway.test.uit.testcase.login.Login_002 -->
    </classes>
  </test> <!-- Login(failed) -->
</suite> <!-- Failed suite [Suite] -->
